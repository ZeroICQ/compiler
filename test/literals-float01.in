1.23
123.
1e3
1e+4
1e-4
1.e5
1.2e-5
1.0e+5
0.001
10e-10
