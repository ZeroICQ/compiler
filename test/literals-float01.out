(1:0)	float literal	[1.230000]	"1.23"
(2:0)	float literal	[123.000000]	"123."
(3:0)	float literal	[1000.000000]	"1e3"
(4:0)	float literal	[10000.000000]	"1e+4"
(5:0)	float literal	[0.000100]	"1e-4"
(6:0)	float literal	[100000.000000]	"1.e5"
(7:0)	float literal	[0.000012]	"1.2e-5"
(8:0)	float literal	[100000.000000]	"1.0e+5"
(9:0)	float literal	[0.001000]	"0.001"
(10:0)	float literal	[0.000000]	"10e-10"
(10:6)	EOF	[]	" "
